### PR TITLE
Marked variables as final which is initialized in declaration or cons…

### PR DESCRIPTION
**Pull request by sundaramrajendran**
…tructor

### CODE REVIEW

File: client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidEditTextTest.java [hunk 1]
```diff
@@ -31,8 +31,8 @@
     
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String textFieldLocator = "com.paypal.selion.pageobjectsdemoapp:id/edit_text";
+    private final String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
+    private final String textFieldLocator = "com.paypal.selion.pageobjectsdemoapp:id/edit_text";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidEditTextTest.java]: Codacy Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120313/issues/source?bid=2928425&fileBranchId=2943483#l35)

File: client/src/test/java/com/paypal/selion/reports/excelreport/ExcelReportTest.java [hunk 2]
```diff
@@ -50,7 +50,7 @@
  */
 public class ExcelReportTest {
 
-    private String excelReportFileName = "dummy_excel_test.xls";
+    private final String excelReportFileName = "dummy_excel_test.xls";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/reports/excelreport/ExcelReportTest.java]: Codacy Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120392/issues/source?bid=2928425&fileBranchId=2943483#l53)

File: client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java [hunk 3]
```diff
@@ -32,9 +32,9 @@
 
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String messageBoxLocator = "android:id/message";
-    private String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String listViewLocator = "com.paypal.selion.pageobjectsdemoapp:id/scroll_view";
+    private final String messageBoxLocator = "android:id/message";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java]: Codacy Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120317/issues/source?bid=2928425&fileBranchId=2943483#l35)

File: client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java [hunk 4]
```diff
@@ -32,9 +32,9 @@
...
+    private final String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java]: Codacy Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120317/issues/source?bid=2928425&fileBranchId=2943483#l36)

File: client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java [hunk 5]
```diff
@@ -32,9 +32,9 @@
...
+    private final String listViewLocator = "com.paypal.selion.pageobjectsdemoapp:id/scroll_view";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java]: Codacy Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120317/issues/source?bid=2928425&fileBranchId=2943483#l37)

File: client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidEditTextTest.java [hunk 6]
```diff
@@ -31,8 +31,8 @@
     
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String textFieldLocator = "com.paypal.selion.pageobjectsdemoapp:id/edit_text";
+    private final String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidEditTextTest.java]: Codacy Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120313/issues/source?bid=2928425&fileBranchId=2943483#l34)

File: client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidEditTextTest.java [hunk 7]
```diff
@@ -31,8 +31,8 @@
     
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String textFieldLocator = "com.paypal.selion.pageobjectsdemoapp:id/edit_text";
+    private static final String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidEditTextTest.java]: This field and others like it in this file should now have an all uppercase name.

File: client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java [hunk 8]
```diff
@@ -32,9 +32,9 @@
 
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String messageBoxLocator = "android:id/message";
-    private String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String listViewLocator = "com.paypal.selion.pageobjectsdemoapp:id/scroll_view";
+    private static final String messageBoxLocator = "android:id/message";
+    private static final String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
+    private static final String listViewLocator = "com.paypal.selion.pageobjectsdemoapp:id/scroll_view";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java]: This field and others like it in this file should now have an all uppercase name.

File: client/src/test/java/com/paypal/selion/reports/excelreport/ExcelReportTest.java [hunk 9]
```diff
@@ -50,7 +50,7 @@
  */
 public class ExcelReportTest {
 
-    private String excelReportFileName = "dummy_excel_test.xls";
+    private static final String excelReportFileName = "dummy_excel_test.xls";
```
Comment (mach6) [inline client/src/test/java/com/paypal/selion/reports/excelreport/ExcelReportTest.java]: This field should now have an all uppercase name.

**Comment by mach6** [comment]
looks good to me otherwise.

**Comment by sundaramrajendran** [comment]
Addressed all the review comments

**Pull request merged**